### ISSUE TITLE
ER diagram visualization

### ISSUE THREAD

**Issue by luwol03** [albingroen/prismabuilder.io#37]
It would be awesome if we could view the schema directly with the schema builder as an ER diagram. There is already a little [web tool](https://prisma-erd.simonknott.de/) and a [prisma generator](https://github.com/keonik/prisma-erd-generator), but its not very nice copying the schema from one place to several places to few times until I'm done with the changes. Maybe this ER diagram could also be shown while adding new fields.

**Comment by yousihy** [comment]
I agree. This tool is amazing, and it is only missing the visual designer
